### PR TITLE
Add GEPR-GEPRC_F722_AIO config

### PR DESCRIPTION
**Pull request by Linjieqiang**

### CODE REVIEW

File: configs/default/GEPR-GEPRC_F722_AIO.config [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+# serial
+serial 1 64 115200 57600 0 115200
+serial 3 8192 115200 57600 0 115200
+serial 4 2 115200 57600 0 115200
+
+# master
+set mag_bustype = I2C
+set mag_i2c_device = 2
+set mag_hardware = NONE
```
Comment (mikeller) [inline configs/default/GEPR-GEPRC_F722_AIO.config:108]: If this setting is used, Betaflight will explicity NOT support the use of compasses on this hardware, and instruct users to buy a new board from a different manufacturer if they want to use a compass.
If you do not want this, please leave this parameter at its default (`AUTO`).

File: configs/default/GEPR-GEPRC_F722_AIO.config [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+set baro_bustype = I2C
+set baro_i2c_device = 2
+set baro_hardware = NONE
```
Comment (mikeller) [inline configs/default/GEPR-GEPRC_F722_AIO.config:111]: If this setting is used, Betaflight will explicity NOT support the use of barometers on this hardware, and instruct users to buy a new board from a different manufacturer if they want to use a barometer.
If you do not want this, please leave this parameter at its default (`AUTO`).

File: configs/default/GEPR-GEPRC_F722_AIO.config [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
+set motor_pwm_protocol = DSHOT600
+set align_board_yaw = 45
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 100
+set beeper_inversion = ON
+set beeper_od = OFF
+set debug_mode = GYRO_SCALED
```
Comment (mikeller) [inline configs/default/GEPR-GEPRC_F722_AIO.config:124]: This should not be set.

**Review by mikeller — CHANGES_REQUESTED** [review]

File: configs/default/GEPR-GEPRC_F722_AIO.config (hunk 1 above)
Comment (Linjieqiang) [reply configs/default/GEPR-GEPRC_F722_AIO.config]: Fixed it.

File: configs/default/GEPR-GEPRC_F722_AIO.config (hunk 3 above)
Comment (Linjieqiang) [reply configs/default/GEPR-GEPRC_F722_AIO.config]: Fixed it.

File: configs/default/GEPR-GEPRC_F722_AIO.config (hunk 2 above)
Comment (Linjieqiang) [reply configs/default/GEPR-GEPRC_F722_AIO.config]: Fixed it.

**Comment by Linjieqiang** [comment]
......

**Comment by Linjieqiang** [comment]
Updated it.

**Review by mikeller — APPROVED** [review]

**Pull request merged**